docs/features/filtering.md

```diff
@@ -175,14 +175,13 @@ resources :people do
   post 'filter', on: :collection
 end
 ```
-
-Then call the `fae_paginate` helper under the table:
+Add a `paginated` class to the items table, then call the `fae_paginate` helper under the table:
 
 `app/views/admin/people/index.html.slim`
 ```slim
 / app/views/admin/people/index.html.slim
 
-table
+table.paginated
```

Comment:
I would like things to work automatically with as little integration steps as possible. Instead of using this class to look up the table, what do you think about not requiring it and looking up the sibling table above `fae_paginate`?

Reply:
Sounds good to me!